fhirstore/fhirstore.py

```diff
@@ -206,3 +222,37 @@ def validate(self, resource):
         if schema is None:
             raise Exception(f"missing schema for resource {resource}")
         validate(instance=resource, schema=schema)
+
+    def search(self, resourceType, params):
+        """
+        Searchs for params inside a resource.
+        Returns a bundle of items, as required by FHIR standards.
+
+        Args:
+            - resource: FHIR resource (eg: 'Patient')
+            - params: search parameters as returned by the API. For a simple
+            search, the parameters should be of the type {"key": "value"}
+            eg: {"gender":"female"}, with modifiers {"address.city:exact":"Paris"}.
+            For a search on multiple parameters, params should be a payload of the form:
+            {"key1": ["value1", "value2"], "multiple": {"key2": ["value3", "value4"]}}.
```

Comment:
What does the `"multiple"` here stands for ?

Reply:
I added some more explanation :)